admiral/pkg/client/clientset/versioned/clientset.go

```diff
@@ -80,11 +105,11 @@ func NewForConfig(c *rest.Config) (*Clientset, error) {
 // NewForConfigOrDie creates a new Clientset for the given config and
 // panics if there is an error in the config.
 func NewForConfigOrDie(c *rest.Config) *Clientset {
-	var cs Clientset
-	cs.admiralV1 = admiralv1.NewForConfigOrDie(c)
-
-	cs.DiscoveryClient = discovery.NewDiscoveryClientForConfigOrDie(c)
-	return &cs
+	cs, err := NewForConfig(c)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Sorry for the second comment on the same line. I think we should return an error to the caller.